README.md

```diff
@@ -32,7 +32,7 @@ Whatsapp Web ([HN thread](https://news.ycombinator.com/item?id=10930718)). Nativ
 Install Nativefier globally with `npm install -g nativefier` . Requirements:
 
 - macOS 10.10+ / Windows / Linux
```

Comment:
Electron 20 removes support for  macOS 10.11 and 10.12 so the minimum version is now macOS 10.13.
https://www.electronjs.org/docs/latest/breaking-changes#removed-macos-1011--1012-support

Reply:
Great catch. Thanks for the attention to @mattruzzi .